recipe/meta.yaml

```diff
@@ -39,12 +39,14 @@ requirements:
     - pybind11 >=2.2.3
     - adios >=1.13.1  # [unix]
     - hdf5  >=1.8.13
+    - blosc
+    - zlib
+    - bzip2
   run:
-    # FIXME: for some reason warnings about this
+    - python
```

Comment:
note to myself: needed to be added here for package variants. Can now be removed again from `test: requires:` section